addon/styles/_pix-input.scss

```diff
@@ -1,7 +1,14 @@
 .pix-input {
   position: relative;
-  display: flex;
+  display: inline-flex;
   flex-direction: column;
+  gap: var(--pix-spacing-1x);
+
+  &--inline {
+    flex-direction: row;
+    gap: var(--pix-spacing-2x);
+    align-items:center;
```

Comment:
```suggestion
    align-items: center;
```

Reply:
le linter, il a rien dit 😢

Reply:
C'est une mission pour moi ça 😂

Reply:
Ah non j'avais oublié que `stylelint` avait dépréciée toutes les règles d'espacement https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules 😭 

Ils conseillent d'utiliser Prettier.